### PR TITLE
TINY-8300: Renamed isChrome to isChromium

### PR DESCRIPTION
**Pull request by HAFRMO**
Related Ticket:

Description of Changes:
Renamed isChrome to isChromium and a lot of Chrome to Chromium

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/` for new features (if applicable)

Review:
* [x] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: modules/sand/src/test/ts/atomic/core/BrowserTest.ts [hunk 1]
```diff
@@ -82,21 +82,21 @@ describe('BrowserTest', () => {
...
   });
 
-  it('Chrome', () => {
-    check('isChrome', 'Windows', 'Chrome', 3, 0, 'Mozilla/5.0 (Windows; U; Windows NT 6.0; en-US) AppleWebKit/532.0 (KHTML, like Gecko) Chrome/3.0.195.38 Safari/532.0');
-    check('isChrome', 'Windows', 'Chrome', 4, 0, 'Mozilla/5.0 (Windows; U; Windows NT 6.0; en-US) AppleWebKit/532.0 (KHTML, like Gecko) Chrome/4.0.195.38 Safari/532.0');
-    check('isChrome', 'Windows', 'Chrome', 5, 0, 'Mozilla/5.0 (Windows; U; Windows NT 6.0; en-US) AppleWebKit/533.4 (KHTML, like Gecko) Chrome/5.0.375.127 Safari/533.4');
-    check('isChrome', 'Windows', 'Chrome', 6, 0, 'Mozilla/5.0 (Windows; U; Windows NT 6.0; en-US) AppleWebKit/534.3 (KHTML, like Gecko) Chrome/6.0.472.53 Safari/534.3');
-    check('isChrome', 'Linux', 'Chrome', 5, 0, 'Mozilla/5.0 (X11; U; Linux x86_64; en-US) AppleWebKit/533.4 (KHTML, like Gecko) Chrome/5.0.375.127 Safari/533.4');
-    check('isChrome', 'macOS', 'Chrome', 6, 0, 'Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_6_4; en-US) AppleWebKit/534.3 (KHTML, like Gecko) Chrome/6.0.472.53 Safari/534.3');
+  it('Chromium', () => {
```
Comment (lnewson) [inline modules/sand/src/test/ts/atomic/core/BrowserTest.ts:85]:
```suggestion
  it('Google Chrome', () => {
```
This should still be `Chrome` (or similar) as all the user agent strings here are from Google Chrome, not Chromium.

File: modules/tinymce/tools/docs/tinymce.Env.js [hunk 2]
```diff
@@ -26,12 +26,12 @@
  */
 
 /**
- * Returns <code>true</code> if the user's browser is Google Chrome.
+ * Returns <code>true</code> if the user's browser is Google Chromium based, such as Google Chrome or Chrome Edge.
```
Comment (lnewson) [inline modules/tinymce/tools/docs/tinymce.Env.js:29]:
```suggestion
 * Returns <code>true</code> if the user's browser is Chromium based, such as Google Chrome or newer versions of Microsoft Edge.
```

File: modules/tinymce/tools/docs/tinymce.Env.js [hunk 3]
```diff
@@ -26,12 +26,12 @@
...
  * <br>
  * <em>Added in TinyMCE 5.1</em>
  *
- * @method browser.isChrome
- * @return {Boolean} Returns <code>true</code> if the user's browser is Google Chrome.
+ * @method browser.isChromium
+ * @return {Boolean} Returns <code>true</code> if the user's browser is Google Chromium based.
```
Comment (lnewson) [inline modules/tinymce/tools/docs/tinymce.Env.js:34]:
```suggestion
 * @return {Boolean} Returns <code>true</code> if the user's browser is Chromium based.
```

**Review by lnewson — APPROVED** [review]

**Comment by lnewson** [comment]
Adding Tyler to review the API doc string changes/suggestions.

**Review by ltrouton — APPROVED** [review]

**Pull request merged**